indico/util/fs_test.py

```diff
@@ -43,13 +43,13 @@ def test_secure_client_filename(filename, expected):
     ('   ', '', 'fallback'),
     ('foo.', '', 'foo'),
     ('foo', '.txt', 'foo'),
-    ('\u4e17', '', 'fallback'),
-    ('\u4e17.txt', '', 'fallback.txt'),
-    ('\u4e17.txt', '.zip', 'fallback.txt'),
-    ('\u4e17.\u4e17', '', 'fallback'),
-    ('\u4e17.', '.txt', 'fallback.txt'),
-    ('\u4e17.\u4e17', '.txt', 'fallback.txt'),
-    ('.\u4e17', '.txt', 'fallback'),
```

Comment:
Please do not fully remove the tests that trigger the fallback, but replace them with some characters that cannot be decoded and then add your new ones in addition to them. `\u4e06` has no representation in unidecode for example. No idea if that's correct or not (since it seems to be a normal CJK character), but it should do the job for now.